bitcoin-magazine/bitcoin-explained/explaining-rgb-tokens-the-technical-side-of-bitcoin.md

```diff
@@ -0,0 +1,1298 @@
+But, you know-
+
+Sjors Provoost: 00:07:12
+
+Yeah, but you couldn't add that to the node, right?
+If, say, we made a change to the Bitcoin Core assets...
+You know what?
+We know that's the Whitepaper.
+Let's just skip those coins.
+
+Ruben Somsen: 00:07:22
+
+I guess theoretically you could throw it out of UTXO sets if you're certain they're un-spendable.
+
+Sjors Provoost: 00:07:27
+
+That's the problem.
+If you're certain, what if some really really smart alien actually made it look like it's just the Whitepaper!
+But no no no, it's really there are private keys for it and one day this alien tries to spend the Whitepaper...
+...and we accidentally soft fork those coins out of existence and this alien gets really angry because he's like, "Hey, you can't just confiscate my money!"
+
+Ruben Somsen: 00:07:48
+
+That sounds like an ECDSA break, but I'm not sure.
+Depends on, I guess, how it's done.
+
+Sjors Provoost: 00:07:53
+
+Maybe the person who created the Whitepaper actually generated lots of public keys from private keys...
+...and then reordered them in such a way that it created a readable Whitepaper.
+
+Ruben Somsen: 00:08:05
+
+Oh yeah.
+
+Sjors Provoost: 00:08:06
+
+I don't think that's even practical.
+
+Ruben Somsen: 00:08:07
+
+No, but to your point, maybe there is a way in which it is actually valid and we can't be 100% certain.
+But one thing I wanted to add-
+
+Aaron van Wirdum: 00:08:14
+
+There's also a thing of where do you draw the line because we do have vanity addresses and you can sort of use those as well to create stuff...
+...and where, at what point are you sure it's a vanity address and at what point (trails off)
+
+Sjors Provoost: 00:08:26
+
+Well, that would be absolutely unacceptable.
+The idea that you can't just decide what is vanity and what should be thrown away.
+
+Aaron van Wirdum: 00:08:33
+
+That's my point, I agree.
+
+Sjors Provoost: 00:08:35
+
+But if you could mathematically prove that a coin cannot be spent, like theoretically impossible to be spent, then you could have that discussion.
+But the problem is we can't really prove that.
+
+Ruben Somsen: 00:08:45
+
+Yeah, but yeah, so one thing that I wanted to add is that it is actually important, that if you want to put data on the Bitcoin blockchain, it needs to be a valid transaction.
+So you can't just, that's why we're talking about these addresses, right?
+Because you can't just take the Bitcoin Whitepaper, just put it there and just be like, "Hey, please put this in the blockchain!"
+No! You need to do it in a format that the blockchain recognizes and that's how you end up with these addresses.
+
+Sjors Provoost: 00:09:09
+
+Yeah okay.
+
+Aaron van Wirdum: 00:09:11
+
+Yeah okay so that was the original problem - people were just bloating the blockchain with all sorts of nonsense.
+Nonsense that most of us don't care about and we still need to all validate it and store it or whatever.
```

Comment:
avoid rephrasings. Here there was no need to finish the last sentence and add "Nonsense" in order to start the new one. It could be:
"That was the original problem, people were just bloating the blockchain with all sorts of nonsense that most of us don't care about and we still need to all validate it and store it or whatever."

Reply:
I did it to break the very long original sentence into two. The second half did not make sense without adding the word Nonsense, which was the last word of the previous sentence.